### PR TITLE
DOCS-7165 made edits

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

**Review by sbehrends — DISMISSED** [review]
LGTM
I believe it tackles the specific feedback of the ticket and also adds some details

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 1]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-A permission called API Manager Environment Administrator gives users full access to API Manager operations inside environments. API Manager environment administrator only relates to actions inside of API Manager.
+To be able to fully access API Manager operations within an environment, users must have the API Manager Environment Administrator permission. An API Manager environment administrator role is associated to tasks performed only in API Manager. You can change permissions for API Manager-related operations from Anypoint Access Management.
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc]:
```suggestion
To be able to fully access API Manager operations within an environment, users must have the API Manager Environment Administrator permission. An API Manager Environment Administrator role is associated with tasks performed only in API Manager. You can change permissions for API Manager-related operations from Anypoint Access Management.
```

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 2]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
...
 
-To change permissions, you need access to the Access Management feature of Anypoint Platform.
+Regarding roles, note that:
 
-API Manager also supports additional environment granular permissions:
+* Roles and permissions assigned to a user in one particular business group are independent of the roles and permissions assigned to that same user in other business groups. 
+* Roles are business group specific, so ensure that you are in the correct business group for which to manage roles by clicking the menu next to your username on the top-right of the screen.
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc]:
```suggestion
* Roles are business group-specific, so ensure that you are in the correct business group for which to manage roles by clicking the menu next to your username on the top-right of the screen.
```

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 3]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
...
 
-* *Deploy API Proxies* - Deploy or redeploy an API proxy - Requires Runtime Manager and
+The following diagram explains the tasks covered by each permission:
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc]:
```suggestion
The following diagram illustrates the tasks enabled by having each permission:
```

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 4]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
...
+
+image::apim-permissions-relationships.png[permission relationships]
+
+In the above diagram, an Environment Administrator has access to all the functionalities and permissions.
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc]:
```suggestion
In this diagram, note that an Environment Administrator has access to all the functionalities and permissions.
```

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 5]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
...
+Outer permissions provide permission to the inner permissions. For example, Manage Alerts gives permission to View Alerts and to View API Configuration.
+
+API Manager supports the following additional environment-level permissions:
+
+[%header,cols="40a,15a,60a"]
+|===
+| Permission | Description | Notes
+| Deploy API Proxies | Deploy or redeploy an API proxy | Requires Runtime Manager and
 Hybrid access. This role enables deployment access from the API Manager, but cannot
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc]:
```suggestion
Hybrid access. This role enables deployment access from API Manager, but cannot
```

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 6]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
...
 guarantee that deployment is successful due to its dependency on Runtime Manager or Hybrid.
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc:27]: "Hybrid"? What is the full name?

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 7]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
...
-* *Manage APIs Configuration* - Import APIs from Exchange. This permission also
-lets a user promote an API to another environment, edit the configuration of an API, and delete
+| Manage Alerts | Create, edit, and delete alerts. 
+| Manage APIs Configuration | Import APIs from Exchange | This permission also
+enables a user to promote an API to another environment, edit the configuration of an API, and delete
 APIs.
 +
 [NOTE]
 --
 This permission is per environment. API visibility can’t be restricted to specific users inside your Organization
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc]:
```suggestion
This permission is per environment. API visibility can’t be restricted to specific users inside your organization.
```

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 8]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
...
 --
-* *Manage Contracts* - Deploy, reject, and revoke contracts. This permission also gives the user permission over tiers.
-* *Manage Policies* - Create, edit, delete, enable, and disable a policy. Also lets a user edit the order of policies.
-* *View Alerts* - List alerts and view configuration summary of an alert.
-* *View APIs Configuration* - List APIs, view configuration summary of an API, and download proxies.
+| Manage Contracts | Deploy, reject, and revoke contracts | This permission also gives the user permission over tiers.
+| Manage Policies | Create, edit, delete, enable, disable, and reorder a policy 
+| View Alerts | List alerts and view configuration summary of an alert 
+| View APIs Configuration | List APIs, view configuration summary of an API, and download proxies.
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc]: These three rows have only two columns?

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 9]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
...
 [NOTE]
 --
 This permission is per environment. API visibility can’t be restricted to specific users inside your Organization
 --
-* *View Contracts* - List contracts and their configuration. Can also list SLA tiers and their configuration.
-* *View Policies* - View a listing of policies and view the configuration of a policy.
-
-These permissions are for an environment, not for an API, and only relate to actions inside API Manager.
+| View Contracts | List contracts and their configuration | Can also list SLA tiers and their configuration.
+| View Policies | View a listing of policies and view the configuration of a policy |
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc:47]: ditto?

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 10]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
...
 
-The relationships of the permissions are as follows:
-
-image::apim-permissions-relationships.png[permission relationships]
 
-An Environment Administrator has access to all functionality and permissions.
-For others, outer permissions provide permission to the inner permissions.
-For example, Manage Alerts gives permission to View Alerts and to View API Configuration.
 
 == Granting Permissions
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc]:
```suggestion
== Grant Permissions
```

Comment (kbostrom) [reply modules/ROOT/pages/environment-permission-task.adoc]: Where is the intro to the steps?

File: modules/ROOT/pages/environment-permission-task.adoc [hunk 11]
```diff
@@ -3,49 +3,54 @@ ifndef::env-site,env-github[]
...
 
-. Click Access Management > Users > _a user to give permissions_ > API Manager > Permissions.
-. Select the permissions for the user and click the blue plus icon image:blue-plus.png[blue plus icon] to save the permissions.
+. *Click Access Management* > *Users* > *_a user to give permissions_* > *API Manager* > Permissions.
+. Select the permissions for the user and click the blue plus icon image:blue-plus.png[blue plus icon] to save the permissions:
```
Comment (kbostrom) [inline modules/ROOT/pages/environment-permission-task.adoc]:
```suggestion
. Select the permissions for the user and click image:blue-plus.png[blue plus icon] to save the permissions:
```

**Review by kbostrom — DISMISSED** [review]
Edited per scope of request

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 4 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 5 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 6 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc:27]: Done! Thanks!

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 7 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 8 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 9 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc:47]: Done! Thanks!

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 10 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc]: Done! Thanks!

File: modules/ROOT/pages/environment-permission-task.adoc (hunk 11 above)
Comment (priyaayyar) [reply modules/ROOT/pages/environment-permission-task.adoc]: Done! Thanks!

**Review by kbostrom — APPROVED** [review]
Reapproval without reedit

**Pull request merged**